components/dashboard/src/components/UsageView.tsx

```diff
@@ -81,6 +90,10 @@ function UsageView({ attributionId }: UsageViewProps) {
         return "Prebuild";
     };
 
+    const getDisplayName = (workspaceClass: string) => {
```

Comment:
This should show the `workspaceClass` as a fallback when the workspace class is no longer defined. The usage records go very far back in time so it's possible a class no longer exists when we're looking at usage history.

Reply:
Ah, good point!

Reply:
Fixed